test/e2e/components/vl-accordion.js

```diff
@@ -6,11 +6,31 @@ class VlAccordion extends VlElement {
   }
 
   async titleText() {
-    return (await this.shadowRoot.findElement(By.css('.vl-accordion__title'))).getText();
+    const element = (await this.getTitleSlotElements())[0];
+    if (element) {
+      return element.getText();
+    } else {
+      const slot = await this._getTitleSlot();
+      return slot.getTextContent();
+    }
+  }
```

Comment:
Dit gaat potentieel problemen geven als je bijvoorbeeld de proza-message component als slot element gaat declareren. Wat we typisch doen bij de e2e testen op webcomponent niveau is de afnemers zelf de slot elementen laten afhandelen aangezien de inhoud van een slot element onbekend is. Veiliger is om een `getTitleSlot` functie te introduceren en het bekomen van de tekst van het slot element verhuizen naar de test.

Reply:
Ik wou dit vermijden omdat dat dan een breaking change zou zijn voor afnemers die nu wel gewoon de attributen gebruiken. 
Eventueel kan ik wel de beide voorzien, om nu ook een publieke getter voor het slot te maken ...

Reply:
Tot op heden hebben we geen afnemers die gebruik maken van een slot, dus voor hen zal er normaal niets breken als we de `titleText` functie behouden, wat zeker de bedoeling was. In deze misschien voorlopig niet nodig. Ik zie dat we dezelfde code toegepast hebben bij vl-alert. Vermoedelijk in de assumptie dat de titel vaak niet meer dan uitsluitend een tekst zal bevatten. Voorlopig goed genoeg als het zo werkt in kader van jullie toepassing.

Reply:
Ik voorzie de beide nu. De methode linkText die nog de title als een string teruggeeft en de methode getTitleSlotElements die alle slotted elements teruggeeft. Ik gebruik ook beide in de E2E tests.